pkg/engine/transform/transform_test.go

```diff
@@ -55,7 +55,7 @@ func TestNormalizeForK8sAddVMASPool(t *testing.T) {
 func TestNormalizeMasterResourcesForVMSSPoolUpgrade(t *testing.T) {
 	RegisterTestingT(t)
 	logger := logrus.New().WithField("testName", "TestNormalizeMasterResourcesForVMSSPoolUpgrade")
-	fileContents, e := ioutil.ReadFile("./transformtestfiles/k8s_template.json")
+	fileContents, e := ioutil.ReadFile("./transformtestfiles/k8s_slb_vmss_template.json")
```

Comment:
The existing UT data input for validating VMSS pool upgrade template filtering doesn't include a VMSS resource :(